extensions/amp-accordion/1.0/test/test-accordion.js

```diff
@@ -459,4 +457,198 @@ describes.sandboxed('Accordion preact component', {}, (env) => {
       expect(content.hidden).to.be.true;
     });
   });
+
+  describe('imperative api', () => {
+    let wrapper;
+    let ref;
+
+    let section1;
+    let section2;
+    let section3;
+
+    describe('multi-expand accordion', () => {
+      beforeEach(() => {
+        ref = Preact.useRef();
```

Comment:
I think in this case, it should be `Preact.createRef()`. `useRef` is only for use inside the components themselves.

Reply:
Thanks, updated!